### PR TITLE
Add Network Connection Initiated By RegAsm.EXE

### PR DESCRIPTION
**Pull request by frack113**
### Summary of the Pull Request

Add a rule to detect regasm.exe network connection.
Limit to SMTP as I can not check FP 

### Changelog

new: Network Connection Initiated By RegAsm.EXE
### Example Log Event

Rule from online sandbox

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/network_connection/net_connection_win_regasm_network_activity.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - attack.defense_evasion
+    - attack.t1218.009
+logsource:
+    category: network_connection
+    product: windows
+detection:
+    selection:
+        Initiated: 'true'
+        Image|endswith: '\regasm.exe'
+        DestinationPort: 587 # SMTP
```
Comment (nasbench) [inline rules/windows/network_connection/net_connection_win_regasm_network_activity.yml:20]: Why on this port?

Comment (frack113) [reply rules/windows/network_connection/net_connection_win_regasm_network_activity.yml:20]: This port was from the AgentTesla in the sandbox.
As I can check FP , I have limited the `DestinationPort` to keep the rule high.
A medium without it is perhaps a better way.

Comment (nasbench) [reply rules/windows/network_connection/net_connection_win_regasm_network_activity.yml:20]: Would need to check some more FP. Before we can merge this as the rule is not very useful at the moment.

Comment (nasbench) [reply rules/windows/network_connection/net_connection_win_regasm_network_activity.yml:20]: Checked out some resources. It seems one of the most common is port 80 and even on some VT results. I will reduce it to medium for now. Once its merged i'll give it a week and check VT results for more accuracy.

File: rules/windows/network_connection/net_connection_win_regasm_network_activity.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+title: Network Connection Initiated By RegAsm.EXE
```
Comment (phantinuss) [inline rules/windows/network_connection/net_connection_win_regasm_network_activity.yml:1]:
```suggestion
title: Network Connection To Public IP Initiated By RegAsm.EXE
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**